README.md

```diff
@@ -31,12 +31,12 @@ documentation.
 | Signal/Component      | Overall Status     |
 | --------------------  | ------------------ |
 | Context               | Beta               |
-| Baggage               | Beta               |
+| Baggage               | RC                 |
```

Comment:
I noticed that we have this struct and all of its fields marked as `pub`. It looks like we would mark them internal. Something to do before marking `Baggage` RC?

https://github.com/open-telemetry/opentelemetry-rust/blob/36c48db5fd974e18ef42dfa54c526c5a851a6ab0/opentelemetry/src/baggage.rs#L457-L466

Reply:
@gruebel I think these fields can be made private? Can you check?

Reply:
https://github.com/open-telemetry/opentelemetry-rust/pull/2843